conans/test/functional/command/export_pkg_test.py

```diff
@@ -22,7 +21,7 @@ def test_dont_touch_server(self):
                             requester_class=None,
                             users={"default": [("lasote", "mypass")]})
 
-        client.save({"conanfile.py": str(TestConanFile("Pkg", "0.1"))})
+        client.save({"conanfile.py": str(GenConanfile().with_name("Pkg").with_version("0.1"))})
```

Comment:
This str() conversion in ``client.save()`` is no longer necessary, it can be dropped.